src/views/virtualmachines/tree/hooks/dragndrop.ts

```diff
@@ -1,39 +1,46 @@
 import { VirtualMachineModel } from 'src/views/dashboard-extensions/utils';
 
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { getLabels, getNamespace } from '@kubevirt-utils/resources/shared';
+import { isEmpty } from '@kubevirt-utils/utils/utils';
 import { k8sPatch } from '@openshift-console/dynamic-plugin-sdk';
 import { TreeViewDataItem } from '@patternfly/react-core';
 import { VM_FOLDER_LABEL } from '@virtualmachines/tree/utils/constants';
 
 import { DRAG_N_DROP_LISTENERS, VALID_DRAG_TARGET_BACKGROUND_COLOR } from './constants';
+import { getVMFromElementID } from './utils';
 
-let draggingVMNamespace: null | string = null;
+let draggingVM: null | V1VirtualMachine = null;
```

Comment:
why are we using let here?

Reply:
becuase we change it when the user starts to drag. Should I use signals?